lib/validates_identity/usage.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class ValidatesIdentity
+  class Usage
```

Comment:
`DslUsage`? WDYT?

Reply:
What about Definition?

Reply:
Sure! Its way better! Makes perfect sense to me! 💥

Reply:
Great! I'll commit it soon.